docs/how-it-works.md

```diff
@@ -5,16 +5,17 @@
 For GitHub-hosted runners, Harden-Runner GitHub Action downloads and installs the StepSecurity Agent.
 
 - The code to monitor file, process, and network activity is in the Agent.
-- The agent is written in Go and is open source at https://github.com/step-security/agent
+- The agent is written in Go and can be found [here](https://github.com/step-security/agent)
```

Comment:
@eromosele-stepsecurity how about we frame it like this: `The community tier agent is open-source and can be found [here](). The enterprise agent tier is closed-source. Both agents are written in Go.`